docs/csharp/language-reference/keywords/using-static.md

```diff
@@ -9,15 +9,15 @@ ms.author: "ronpet"
 ---
 # using static Directive (C# Reference)
 
-The `using static` directive designates a type whose static members you can access without specifying a type name. Its syntax is:
+The `using static` directive designates a type whose static members and nested types you can access without specifying a type name. Its syntax is:
 
 ```csharp
-using static <fully-qualified-type-name>
+using static <fully-qualified-type-name>;
 ```
 
-where *fully-qualified-type-name* is the name of the type whose static members can be referenced without specifying a type name. If you do not provide a fully qualified type name (the full namespace name along with the type name), C# generates compiler error CS0246: "The type or namespace name '<type-name>' could not be found."
+where *fully-qualified-type-name* is the name of the type whose static members and nested types can be referenced without specifying a type name. If you do not provide a fully qualified type name (the full namespace name along with the type name), C# generates compiler error [CS0246](../compiler-messages/cs0246.md).
```

Comment:
I like the addition of the link, but the sentence "The type of namespace … could not be found." should not be deleted.

We don't want to repeat all the information in another article, but the one sentence means that readers don't need to click the link for the most basic information.